pkg/vulnsrc/rootio/rootio.go

```diff
@@ -211,9 +222,6 @@ func (vs VulnSrcGetter) Get(params db.GetParams) ([]types.Advisory, error) {
 	}
 
 	rootAdvs := lo.SliceToMap(advs, func(adv types.Advisory) (string, types.Advisory) {
-		// Debian may contain severity in the advisory.
-		// We need to save this severity for the Root.io advisory.
-		adv.Severity = allAdvs[adv.VulnerabilityID].Severity
```

Comment:
You checked the `cveInfo.Severity` field:
https://github.com/urimils/trivy-db/blob/47a5bb5398d4cd1d7e551d7f1e125c27f5f5537c/pkg/vulnsrc/rootio/rootio.go#L104

Is it possible that an advisory doesn’t contain severity info?
That seems possible.

So if the Root.io feed doesn’t include severity but Debian does, should we use Debian’s severity?
Or do you prefer to use our severity selection logic?
(https://trivy.dev/docs/latest/guide/scanner/vulnerability/#severity-selection)

Reply:
Just to confirm that I understood you correctly:
Root.io will include Debian severity (or severity from other sources) in their own feed, so we don’t need to add Debian severity on our side anymore (as it’s currently done in your code), right?